.github/workflows/weekly-pulumi-update.yml

```diff
@@ -40,9 +40,8 @@ jobs:
           cd provider
           go get github.com/pulumi/pulumi/pkg/v3
           go get github.com/pulumi/pulumi/sdk/v3
-          go mod download
-          go mod tidy
           cd ..
+          make ensure
```

Comment:
`ensure` doesn't seem to run the `go mod download` - does this matter? Also, is there benefit to running the `go mod tidy` on the sdk and tests folders - would they have changed?

Reply:
`go mod tidy` implicitly runs `go mod download`

The sdk folder shouldn't have changed, but the test mod pulls in changes from the provider mod

Reply:
I expect we'll clean all this up across the native providers soon anyway, so this is mostly to unblock the failing GHA.